Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxyFunctionsGenerated.cpp

```diff
@@ -3047,17 +3048,25 @@ void RemoteGraphicsContextGLProxy::setDrawingBufferColorSpace(const WebCore::Des
     }
 }
 
+void RemoteGraphicsContextGLProxy::drawingBufferToPixelBuffer(WebCore::GraphicsContextGLFlipY, RefPtr<WebCore::PixelBuffer>&&)
+{
+    RELEASE_ASSERT_NOT_REACHED();
```

Comment:
This is a symptom of doing something incorrect. 

E.g. interface has two functions and two backends. One of the functions is invalid for each backend.

Reply:
Also, the ffile is Generated, you cannot go around modifying it by hand.